cmd/sqyrrl.go

```diff
@@ -118,18 +118,78 @@ func printHelp(cmd *cobra.Command, args []string) {
 	}
 }
 
-func startServer(cmd *cobra.Command, args []string) error {
+func startServer(cmd *cobra.Command, args []string) (err error) { // NRV
 	logger := configureRootLogger(&cliFlagsSelected)
 
-	return server.ConfigureAndStart(logger, server.Config{
-		Host:          cliFlagsSelected.host,
-		Port:          cliFlagsSelected.port,
-		CertFilePath:  cliFlagsSelected.certFilePath,
-		KeyFilePath:   cliFlagsSelected.keyFilePath,
-		EnvFilePath:   cliFlagsSelected.envFilePath,
-		EnableOIDC:    cliFlagsSelected.enableOIDC,
-		IndexInterval: cliFlagsSelected.indexInterval,
-	})
+	var config server.Config
+	if cliFlagsSelected.configFilePath != "" {
+		var tomlData []byte
+		if tomlData, err = os.ReadFile(cliFlagsSelected.configFilePath); err != nil {
+			return err
+		}
+
+		_, err = toml.Decode(string(tomlData), config)
```

Comment:
```suggestion
		_, err = toml.Decode(string(tomlData), &config)
```
I'm seeing
```
Error: toml: cannot decode to non-pointer "server.Config"
```
when I try a linux amd64 binary. Seems to work ok with this fix.